### DISCUSSION TITLE
Source for F4 blue bill?

### DISCUSSION THREAD

**Discussion by cube1us** [erichelgeson/BlueSCSI-v1#48] [Q&A] [answered]
It is not clear that the current source supports the F4, though I see at least one indication it might (a branch labelled F1 Stale).  Is the current code for the F4 (which is what I got from TbA) or the F1?  It the current code is just for the F1, where do I get the F4 code (or how do I tweak it for F4)?

I really need CD support, similar to what is found at https://github.com/mynameistroy/BlueSCSI

JRJ

**Comment by cube1us** [comment]
Well, I have an answer, now.  The version from Tech By Androda IS NOT OPEN SOURCE.  Frankly, I feel deceived a bit, because NOWHERE in the links from this github to blue.scsi to the TbA page mention the fact that THAT one is NOT open source.  At the top of the blue.scsi page it reads "BlueSCSI is an open source, open hardware, and *open * design SCSI solution"  With this situation, I suggest that there SHOULD NOT BE A LINK FROM blue.scsi to the TbA page - the current situation is DECEPTIVE.

**Reply by erichelgeson** [reply]
I agree and am changing it now - it wasn't my intent to deceive and I will be including a page clarifying the license of each fork. Unfortunately open hardware and software is ripe with people looking to take your work and make a quick buck - so we're trying to strike a balance where this project is not just commercialized and stays open for all of us. The F4 fork will eventually be open sourced so if you're uncomfortable with the current license please wait till it is open. 

I'd rather be doing anything than think about licenses and holding off cloners, but such is the current situation.

**Reply by cube1us** [reply]
Thanks for the update.  It was on his web page, but as I went back and forth about what to buy, I must have forgotten about it. 

Good to know that eventually there will be an F4 fork.

**Comment by erichelgeson** [comment, marked as answer]
It has since been opensourced here https://github.com/androda/F4_BlueSCSI
